tests/benchmark-runner-tests.mjs

```diff
@@ -1,7 +1,6 @@
-import { BenchmarkRunner } from "../resources/benchmark-runner.mjs";
+import {BenchmarkRunner} from "../resources/benchmark-runner.mjs";
```

Comment:
I think we want to keep a space after { & before }.

Reply:
awesome - agreed - it's more readable. 
I'll adjust the rule!